memory/main.py

```diff
@@ -400,7 +396,7 @@ def click_to_control_dumb():
     logger.debug("Awaiting control (clicking)")
     with logging_redirect_tqdm():
         fmt = "Awaiting control... elapsed {elapsed}"
-        with tqdm(bar_format=fmt) as pbar:
+        with tqdm(bar_format=fmt):
```

Comment:
Do we need this at all? I could be mistaken, but as I understand we need to use `pbar` in order to make use of this. @orkaboy would know better. They added this to begin with